src/libutil/args.cc

```diff
@@ -39,7 +39,7 @@ void Completions::add(std::string completion, std::string description)
 bool Completion::operator<(const Completion & other) const
 { return completion < other.completion || (completion == other.completion && description < other.description); }
 
-bool pathCompletions = false;
+std::string completionType = "normal";
```

Comment:
Maybe it's better to change this to an enum.

Reply:
Done